core/modules/comment/comment.install

```diff
@@ -364,6 +364,75 @@ function comment_update_1003() {
 }
 
 /**
+ * Replace any "Recent comments" blocks previously provided by comment.module
+ * with blocks provided by block displays of the "Recent comments" view.
+ */
+function comment_update_1004() {
+
+  // Enable the "Recent comments" view if not already enabled.
+  $view = views_get_view('comments_recent');
+  if (!views_view_is_enabled($view)) {
+    $view->disabled = FALSE;
+    $view->save();
+  }
+
+  $layouts = layout_load_all();
```

Comment:
Similar to the comment about loading a view, we need to only use config calls to load the layouts. We can get a list of all layouts with `config_get_names_with_prefix('layout.layout.')`, then loop through loading them with `config()`.